### PR TITLE
Feature/3601  default space role per space

### PR DESCRIPTION
**Pull request by asf-algoo**
related to #3601 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by grignards — APPROVED** [review]

File: backend/tracim_backend/views/core_api/schemas.py [hunk 1]
```diff
@@ -965,6 +966,14 @@ class WorkspaceModifySchema(marshmallow.Schema):
         default=None,
         allow_none=True,
     )
+    default_user_role = StrippedString(
+        example=WorkspaceRoles.READER.slug,
+        description="default role for new user in this workspace",
```
Comment (raphj) [inline backend/tracim_backend/views/core_api/schemas.py:971]:
```suggestion
        description="default role for new users in this workspace",
```

File: backend/tracim_backend/views/core_api/schemas.py [hunk 2]
```diff
@@ -984,6 +993,12 @@ class WorkspaceCreationSchema(marshmallow.Schema):
         validate=workspace_access_type_validator,
         required=True,
     )
+    default_user_role = StrippedString(
+        description="default role for new user in this workspace",
```
Comment (raphj) [inline backend/tracim_backend/views/core_api/schemas.py:997]:
```suggestion
        description="default role for new users in this workspace",
```

File: backend/tracim_backend/views/core_api/schemas.py [hunk 3]
```diff
@@ -1060,6 +1075,12 @@ class Meta:
 
 
 class WorkspaceSchema(WorkspaceDigestSchema):
+    default_user_role = StrippedString(
+        example=WorkspaceRoles.READER.slug,
+        validate=user_role_validator,
+        required=True,
+        description="default role for new user in this workspace",
```
Comment (raphj) [inline backend/tracim_backend/views/core_api/schemas.py:1082]:
```suggestion
        description="default role for new users in this workspace",
```

**Review by raphj — APPROVED** [review]

**Review by raphj — APPROVED** [review]

**Comment by grignards** [comment]
Tested on branch.

**Pull request merged**